### ISSUE TITLE
Text to speech

### ISSUE THREAD

**Issue by greatislander** [accessibility-exchange/platform#497]

**Comment by jobara** [comment]
In conversation today with @cherylhjli, @greatislander, and @michelled:

- use case is to help with comprehension, not to replace a screen reader for example.
- Needs to be read
  - main content
  - navigation and other controls
  - does not need to read back the icons (i.e. doesn't have to read the aria-label)
- For a first pass we will try to include the Orator component from Infusion. (only including the relevant JS on a page when the feature is enabled, to minimize the performance impact).
  - Page reading will read back main content
  - Highlighting to read, should be able to work on any text including in navigation.
  - @cherylhjli will provide designs for the controls.
